apps/medon-fe/src/pages/ResendConfirmation/index.tsx

```diff
@@ -18,10 +18,13 @@ import {
 } from 'pages/ResendConfirmation/styles';
 import { SubmitSendEmail } from 'pages/ResendConfirmation/types';
 import { emailSchema } from 'validation/accountConfirmationSchema';
+import { useResendEmailMutation } from 'redux/api/authApi';
+import { toast } from 'react-toastify';
```

Comment:
Order of import 

node_modules
absolute path

Reply:
changed